test/e2e/specs/editor/editor__post-basic-flow.ts

```diff
@@ -54,8 +55,14 @@ describe( DataHelper.createSuiteTitle( 'Editor: Basic Post Flow' ), function ()
 			await editorPage.enterTitle( title );
 		} );
 
-		it( 'Enter post text', async function () {
-			await editorPage.enterText( quote );
+		it( 'Enter post content', async function () {
+			const blockHandle = await editorPage.addBlockFromSidebar(
```

Comment:
It was this action, which is used in the [`test/e2e/specs/editor/editor__post-advanced-flow.ts`](https://github.com/Automattic/wp-calypso/blob/trunk/test/e2e/specs/editor/editor__post-advanced-flow.ts) spec, which is **not** run for Jetpack deployment E2E.